usersession/userd/settings.go

```diff
@@ -77,15 +94,116 @@ func allowedSetting(setting string) bool {
 	return snap.ValidAppName(base)
 }
 
+func settingWithSub(setting string, subproperty string) string {
+	return fmt.Sprintf("%s/%s", setting, subproperty)
+}
+
+func settingForLog(setting string) string {
+	settingParts := strings.SplitN(string(setting), "/", 2)
+	if len(settingParts) > 1 {
+		return fmt.Sprintf("%q subproperty %q", settingParts[0], settingParts[1])
+	} else {
+		return fmt.Sprintf("%q", settingParts[0])
+	}
+}
```

Comment:
AFAICT, there are no spread tests specifically for this. Am I missing something?

Reply:
There was, see "Ensure settings value validation works", and I just now added a test for the subproperty case also here de73b23f4